### PR TITLE
Docker setup + flake8 tests for students code

### PR DESCRIPTION
**Pull request by gal432**

### CODE REVIEW

File: devops/lms.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+version: "3"
+
+services:
+  db:
+    image: postgres:9.6-alpine
```
Comment (yammesicka) [inline devops/lms.yaml]: 🐱

File: lmsdb/models.py [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
     ForeignKeyField,
     IntegerField,
     ManyToManyField,
-    PostgresqlDatabase,
-    SqliteDatabase,
-    TextField,
-)
-from playhouse.signals import Model, pre_save  # type: ignore
+    TextField)
+
```
Comment (yammesicka) [inline lmsdb/models.py]: Remove extra line

File: lmsdb/models.py [hunk 3]
```diff
@@ -1,27 +1,28 @@
...
+from playhouse.signals import Model, pre_save
+
```
Comment (yammesicka) [inline lmsdb/models.py]: Remove extra line

File: lmslinter/flake8/services.py [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+FLAKE_SKIP_ERRORS = (
+    'T001',  # print found
+    'W292',  # no new line in the end of the code
+)
+
+
+class PyFlakeChecker:
+    def __init__(self, solution_check_pk: str, logger):
```
Comment (yammesicka) [inline lmslinter/flake8/services.py]: type for logger?

File: tests/conftest.py [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import datetime
 
-from lms.lmsweb.models import (
-    ALL_MODELS, Role, RoleOptions, User, Exercise, CommentText,
+from lmsdb.models import (
```
Comment (yammesicka) [inline tests/conftest.py]: `lms.`

File: tests/conftest.py [hunk 6]
```diff
@@ -1,7 +1,7 @@
...
+    ALL_MODELS, Role, RoleOptions, User, Exercise, CommentText, Solution
```
Comment (yammesicka) [inline tests/conftest.py]: Add trailing ,

File: tests/test_models.py [hunk 7]
```diff
@@ -1,4 +1,4 @@
-from lms.lmsweb.models import User
+from lmsdb.models import User
```
Comment (yammesicka) [inline tests/test_models.py]: `lms.`

File: lmslinter/flake8/tasks.py [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+from lmslinter.config.celery import app
+from lmslinter.flake8 import services
+
+
+_logger: logging.Logger = get_task_logger(__name__)
+_logger.setLevel(logging.INFO)
+
+
+@app.task
+def run_flake8_on_solution(solution_pk):
```
Comment (yammesicka) [inline lmslinter/flake8/tasks.py]: typoe?

File: lmslinter/flake8/tasks.py [hunk 9]
```diff
@@ -0,0 +1,21 @@
+import logging
+
+from celery.utils.log import get_task_logger
+
+from lmslinter.config.celery import app
```
Comment (yammesicka) [inline lmslinter/flake8/tasks.py]: lms.

File: lmsweb/__init__.py [hunk 10]
```diff
@@ -18,4 +18,8 @@
 csrf = CSRFProtect(webapp)
 
 # Must import files after app's creation
-from lms.lmsweb import models, views  # NOQA: F401
+from lmsdb import models  # NOQA: F401
+from lmsweb import views  # NOQA: F401
+
+
+application = webapp
```
Comment (yammesicka) [inline lmsweb/__init__.py]: ?

File: lmslinter/flake8/services.py [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+            self._logger.info('Adding error %s to solution %s',
+                              error, self.solution_id)
+            text = FLAKE_ERRORS_MAPPING.get(
+                error.error_code, f'{error.error_code}-{error.text}')
+            comment, _ = models.CommentText.get_or_create(text=text)
+            models.Comment.create(
+                commenter=models.User.get_system_user(),
+                line_number=error.line_number,
+                comment=comment,
+                solution=self.solution)
```
Comment (yammesicka) [inline lmslinter/flake8/services.py]: `,`, close parenthesis next line

File: lmsdb/models.py (hunk 2 above)
Comment (gal432) [reply lmsdb/models.py]: flake8

File: lmsdb/models.py (hunk 3 above)
Comment (gal432) [reply lmsdb/models.py]: flake8

File: lms/lmsdb/models.py [hunk 12]
```diff
@@ -1,27 +1,30 @@
...
     IntegerField,
     ManyToManyField,
-    PostgresqlDatabase,
-    SqliteDatabase,
     TextField,
 )
-from playhouse.signals import Model, pre_save  # type: ignore
+
+from playhouse.signals import Model, pre_save
+
```
Comment (yammesicka) [inline lms/lmsdb/models.py]: Remove extra line

File: lms/lmsdb/models.py [hunk 13]
```diff
@@ -1,27 +1,30 @@
...
 import secrets
 import string
 from datetime import datetime
 
-from flask_admin import Admin, AdminIndexView  # type: ignore
-from flask_admin.contrib.peewee import ModelView  # type: ignore
-from flask_login import UserMixin, current_user  # type: ignore
-from peewee import (  # type: ignore
+from flask_login import UserMixin
+
```
Comment (yammesicka) [inline lms/lmsdb/models.py]: Remove extra line

**Comment by yammesicka** [comment]
Great job @gal432, awesome addition to our project.

**Pull request merged**